public/javascripts/SVLabel/src/SVLabel/label/LabelContainer.js

```diff
@@ -97,7 +166,8 @@ function LabelContainer($) {
 
     //remove old versions of this label, add updated label
     this.addUpdatedLabel = function (tempId) {
-        var otherLabels = _.filter(currentCanvasLabels,
+        // fix for object version
```

Comment:
What does this mean?

Reply:
ah just an artifact from when i was implementing, I'll get rid of it